packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1255,4 +1262,40 @@ void main() {
     expect(find.text('C'), findsOneWidget);
     expect(find.text('D'), findsNothing);
   });
+
+  testWidgets('Correct custom expanded padding', (WidgetTester tester) async {
+    const Key firstPanelKey = Key('firstPanelKey');
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SingleChildScrollView(
+          child: SimpleExpansionPanelListTestWidget(
+            firstPanelKey: firstPanelKey,
+            canTapOnHeader: true,
+            expandedHeaderPadding: EdgeInsets.symmetric(vertical: 40.0),
+          ),
+        ),
+      ),
+    );
+
+    // The panel is closed
+    expect(find.text('A'), findsOneWidget);
+    expect(find.text('B'), findsNothing);
+    RenderBox box = tester.renderObject(find.byType(ExpansionPanelList));
+    final double oldHeight = box.size.height;
+    expect(box.size.height, equals(oldHeight));
+
+    // Now, expand the child panel.
+    await tester.tap(find.byKey(firstPanelKey));
+    await tester.pumpAndSettle();
+
+    // The panel is expanded
+    expect(find.text('A'), findsNothing);
+    expect(find.text('B'), findsOneWidget);
+    box = tester.renderObject(find.byType(ExpansionPanelList));
+
+    // Padding is added to top and bottom of expanded header (subtracting body height)
+    final double heightDelta = box.size.height - oldHeight - 107.0;
+    expect(heightDelta, equals(80.0)); // Double of symmetric padding value
+  });
```

Comment:
In pseudocode, the strategy I would suggest here is something along the lines of:
1. Create an expansion panel with custom header padding.
2. Check that there is no padding when the panel is closed
3. Open the panel (or if another panel is already open, just examine that one. It seems that SimpleExpansionPanelListTestWidget defaults to having one open and one closed)
4. Check that the custom padding is used when the panel is open

Reply:
Alright, test has been adjusted + new test added for testing the default padding.